### PR TITLE
#302/unit tests

### PR DESCRIPTION
**Pull request by Hambrsoom**
I added some new unit tests to increase our testing coverage a bit.

image

### CODE REVIEW

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/InfoCardVanierLibraryTest.java [hunk 1]
```diff
@@ -54,7 +54,7 @@
             GrantPermissionRule.grant(
                     "android.permission.ACCESS_FINE_LOCATION",
                     "android.permission.ACCESS_COARSE_LOCATION",
-                    "android.permission.WRITE_CALENDAR",
+                    "android.permissigit on.WRITE_CALENDAR",
```
Comment (thomasgauvin) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/InfoCardVanierLibraryTest.java:57]: wat

**Comment by ferdousara-parvin** [comment]
Looks good!

File: app/src/androidTest/java/com/example/concordia_campus_guide/activities/InfoCardVanierLibraryTest.java (hunk 1 above)
Comment (thomasgauvin) [inline app/src/androidTest/java/com/example/concordia_campus_guide/activities/InfoCardVanierLibraryTest.java:57]:
```suggestion
                    "android.permission.WRITE_CALENDAR",
```

Comment (Hambrsoom) [reply app/src/androidTest/java/com/example/concordia_campus_guide/activities/InfoCardVanierLibraryTest.java:57]: I don't even know how this happened. I will fix them immediately

Comment (Hambrsoom) [reply app/src/androidTest/java/com/example/concordia_campus_guide/activities/InfoCardVanierLibraryTest.java:57]: on it

**Pull request merged**